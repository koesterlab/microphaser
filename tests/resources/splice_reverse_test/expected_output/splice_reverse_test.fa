>d30b9dac24c00d9R
AAGCCAAGAAATACTGAAGaAACTATT
>c225e293273e48fR
CCAAGAAATACTGAAGaAACTATTCAG
>7011a59454dda74R
AGAAATACTGAAGaAACTATTCAGGTT
>1efa508d42bd9bcR
AATACTGAAGaAACTATTCAGGTTCTT
>1fca23b4f4cddc8R
ACTGAAGaAACTATTCAGGTTCTTACT
>210cb330fa6bccfR
GAAGaAACTATTCAGGTTCTTACTATA
>302a1fb3856f228R
GaAACTATTCAGGTTCTTACTATAATA
>4f610280fdd88a2R
AGGAAGCCAAGAAATACTGAAGaAACT
>96db41696eab759R
AAAAGGAAGCCAAGAAATACTGAAGaA
>2df200b2e3eb7d3R
AAGCCAAGAAATACTGAAGaAACTATT
>6f698c636022ea8R
CCAAGAAATACTGAAGaAACTATTCTG
>5de706d492a4e33R
AGAAATACTGAAGaAACTATTCTGAAC
>8420c14e9383865R
AATACTGAAGaAACTATTCTGAACACT
>997cb4e940b6d5fR
ACTGAAGaAACTATTCTGAACACTGAT
>24362f4a2ca7fc6R
GAAGaAACTATTCTGAACACTGATGGA
>434939c4c42b55cR
GaAACTATTCTGAACACTGATGGACTT
>84196c0e1674f1aR
AGGAAGCCAAGAAATACTGAAGaAACT
>4a1503929f4884cR
AAAAGGAAGCCAAGAAATACTGAAGaA
