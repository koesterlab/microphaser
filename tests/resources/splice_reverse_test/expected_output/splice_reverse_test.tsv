id	transcript	gene_id	gene_name	chrom	offset	freq	depth	nvar	nsomatic	nvariant_sites	nsomvariant_sites	strand	variant_sites	somatic_positions	somatic_aa_change	germline_positions	germline_aa_change	normal_sequence	mutant_sequence
d30b9dac24c00d9R	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231625	1.0	16	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			AAGCCAAGAAATACTGAAGGAACTATT	AAGCCAAGAAATACTGAAGaAACTATT
c225e293273e48fR	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231628	1.0	16	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			CCAAGAAATACTGAAGGAACTATTCAG	CCAAGAAATACTGAAGaAACTATTCAG
7011a59454dda74R	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231631	1.0	16	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			AGAAATACTGAAGGAACTATTCAGGTT	AGAAATACTGAAGaAACTATTCAGGTT
1efa508d42bd9bcR	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231634	1.0	16	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			AATACTGAAGGAACTATTCAGGTTCTT	AATACTGAAGaAACTATTCAGGTTCTT
1fca23b4f4cddc8R	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231637	1.0	16	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			ACTGAAGGAACTATTCAGGTTCTTACT	ACTGAAGaAACTATTCAGGTTCTTACT
210cb330fa6bccfR	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231640	1.0	16	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			GAAGGAACTATTCAGGTTCTTACTATA	GAAGaAACTATTCAGGTTCTTACTATA
302a1fb3856f228R	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231643	1.0	16	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			GGAACTATTCAGGTTCTTACTATAATA	GaAACTATTCAGGTTCTTACTATAATA
4f610280fdd88a2R	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231622	1.0	16	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			AGGAAGCCAAGAAATACTGAAGGAACT	AGGAAGCCAAGAAATACTGAAGaAACT
96db41696eab759R	ENST00000275053	ENSG00000146263	MMS22L	chr6	97231619	1.0	17	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			AAAAGGAAGCCAAGAAATACTGAAGGA	AAAAGGAAGCCAAGAAATACTGAAGaA
2df200b2e3eb7d3R	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231625	1.0	16	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			AAGCCAAGAAATACTGAAGGAACTATT	AAGCCAAGAAATACTGAAGaAACTATT
6f698c636022ea8R	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231628	1.0	16	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			CCAAGAAATACTGAAGGAACTATTCTG	CCAAGAAATACTGAAGaAACTATTCTG
5de706d492a4e33R	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231631	1.0	16	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			AGAAATACTGAAGGAACTATTCTGAAC	AGAAATACTGAAGaAACTATTCTGAAC
8420c14e9383865R	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231634	1.0	16	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			AATACTGAAGGAACTATTCTGAACACT	AATACTGAAGaAACTATTCTGAACACT
997cb4e940b6d5fR	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231637	1.0	16	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			ACTGAAGGAACTATTCTGAACACTGAT	ACTGAAGaAACTATTCTGAACACTGAT
24362f4a2ca7fc6R	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231640	1.0	16	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			GAAGGAACTATTCTGAACACTGATGGA	GAAGaAACTATTCTGAACACTGATGGA
434939c4c42b55cR	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231643	1.0	16	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			GGAACTATTCTGAACACTGATGGACTT	GaAACTATTCTGAACACTGATGGACTT
84196c0e1674f1aR	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231622	1.0	16	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			AGGAAGCCAAGAAATACTGAAGGAACT	AGGAAGCCAAGAAATACTGAAGaAACT
4a1503929f4884cR	ENST00000369251	ENSG00000146263	MMS22L	chr6	97231619	1.0	17	1	1	1	1	Reverse	97231644	97231644	p.Ser437Phe			AAAAGGAAGCCAAGAAATACTGAAGGA	AAAAGGAAGCCAAGAAATACTGAAGaA
